pkg/apiserver/api_installer.go

```diff
@@ -148,6 +171,19 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage RESTStorage
 
 		itemPath := path + "/{name}"
 		nameParams := append(params, nameParam)
+		namespaceFn = func(req *restful.Request) (namespace string, err error) {
+			return
```

Comment:
I'm still a Go newbie, I guess. What does this do? I don't see a definition of namespace.